internal/usage/usage.go

```diff
@@ -30,6 +30,7 @@ const (
 	DiskSizeGB               = "Capacity, in gigabytes, of the host's root volume."
 	Backup                   = "If true, enables Continuous Cloud Backup for your deployment. This option is unavailable for clusters smaller than M10."
 	BIConnector              = "If true, enables BI Connector for Atlas on the deployment."
+	TerminationProtect       = "If true, enable termination protection for your cluster."
```

Comment:
Actually, one other question: they just have to set the flag, not flag + true, right? (i.e., they specify "--terminationProtect", not "--terminationProtect true")? If so, let's say this for clarity. I'll change the other examples on this page that also use "if true":
```suggestion
	TerminationProtect       = "Enables termination protection for your cluster. You can't delete the cluster with termination protection enabled."
```

Reply:
Yes, the user will just have to set the flag with `--terminationProtect`. I'll update the docs

Reply:
We could also change it from "If true" to "If set"?

Reply:
Either is fine but I think the "if set" is implied if you just say "enables"

Reply:
Done